classes/profile.php

```diff
@@ -269,6 +281,14 @@ public function mainGrade()
             return UserFilter::GRADE_MST;
           case 'D':
             return UserFilter::GRADE_PHD;
+          case 'B':
+            return UserFilter::GRADE_BAC;
+          case 'E':
+            return UserFilter::GRADE_EXE;
+          case 'G':
+            return UserFilter::GRADE_GRD;
+          case 'S':
+            return UserFilter::GRADE_MSP;
```

Comment:
Shouldn't we use readable enum symbols instead, e.g ``GRADE_MASTER`` / ``GRADE_GRADUATE``?